### DISCUSSION TITLE
Scheduled Status

### DISCUSSION THREAD

**Discussion by MichaelJPDX** [neet/masto.js#1030] [Q&A]
I'm trying to schedule a status by passing a scheduled_at value in the parameters of masto.v1.statuses.create() but it isn't working. There's no example of how to do this and the documentation only seems to reference getting already scheduled statuses. Can we schedule statuses using Masto.js or is it not supported?

**Comment by neet** [comment]
Hi @MichaelJPDX, sorry for the inconvenience. You need to pass **`scheduledAt`** in camelCase instead of `scheduled_at` then it should work without any problem. Here are some examples:

```ts
let client = createRestAPIClient({
  url: "https://mastodon.social",
  accessToken: process.env.MASTODON_TOKEN,
});

// create
let status = await client.v1.statuses.create({
  status: "Hello, world!",
  visibility: "public",
  scheduledAt: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(),
});

// fetch
status = await client.v1.scheduledStatuses.$select(status.id).fetch();
// list
let scheduledStatuses = await client.v1.scheduledStatuses.$select(status.id).list();
// delete
await client.v1.scheduledStatuses.$select(status.id).remove();
```

Due to the limited bandwidth, we're currently citing all the docs for the REST API from official website of Mastodon (https://docs.joinmastodon.org) and because of this, the naming convention doesn't match with the actual APIs provided by Masto.js in some cases.

We are aware of this issue and plan to have better documentation in place in the future!
